common/database/database_update_manifest.cpp

```diff
@@ -5468,7 +5468,61 @@ DROP PRIMARY KEY,
 ADD PRIMARY KEY (`id`) USING BTREE,
 ADD INDEX `level_skill_cap`(`skill_id`, `class_id`, `level`, `cap`);
 )"
-	}
+	},
+	ManifestEntry {
+	.version = 9269,
+	.description = "2024_03_10_parcel_implementation.sql",
+	.check = "SHOW TABLES LIKE 'parcels'",
+	.condition = "empty",
+	.match = "",
+	.sql = R"(CREATE TABLE `parcels` (
+				`id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+				`item_id` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+				`slot_id` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+				`quantity` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+				`to_name` VARCHAR(64) NULL DEFAULT NULL COLLATE 'latin1_swedish_ci',
+				`from_name` VARCHAR(64) NULL DEFAULT NULL COLLATE 'latin1_swedish_ci',
+				`note` VARCHAR(1024) NULL DEFAULT NULL COLLATE 'latin1_swedish_ci',
+				`sent_date` DATETIME NULL DEFAULT NULL,
+				PRIMARY KEY (`id`) USING BTREE,
+				UNIQUE INDEX `data_constraint` (`slot_id`, `to_name`) USING BTREE,
+				FULLTEXT INDEX `to_name_search` (`to_name`)
+				)
+				COLLATE='latin1_swedish_ci'
+				ENGINE=InnoDB
+				AUTO_INCREMENT=1;
+				CREATE TABLE `parcel_merchants` (
+					`id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+					`merchant_id` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+					`last_name` VARCHAR(50) NULL DEFAULT NULL COLLATE 'latin1_swedish_ci',
+					PRIMARY KEY (`id`) USING BTREE
+				)
+				COLLATE='latin1_swedish_ci'
+				ENGINE=InnoDB
+				AUTO_INCREMENT=1;
+				REPLACE INTO `parcel_merchants` (`merchant_id`, `last_name`) VALUES 
+				(202129, 'Parcels and General Supplies'),
+				(3036, 'Parcels and General Supplies'),
+				(394025, 'Parcels and General Supplies'),
+				(75113, 'Parcels and General Supplies'),
+				(49073, 'Parcels and General Supplies'),
+				(41021, 'Parcels and General Supplies'),
+				(40070, 'Parcels and General Supplies'),
+				(106115, 'Parcels and General Supplies'),
+				(55150, 'Parcels and General Supplies'),
+				(9053, 'Parcels and General Supplies'),
+				(382156, 'Parcels and General Supplies'),
+				(1032, 'Parcels and General Supplies'),
+				(155088, 'Parcels and General Supplies'),
+				(23017, 'Parcels and General Supplies'),
+				(61065, 'Parcels and General Supplies'),
+				(29008, 'Parcels and General Supplies'),
+				(67058, 'Parcels and General Supplies'),
+				(54067, 'Parcels and General Supplies'),
+				(19031, 'Parcels and General Supplies'),
+				(50140, 'Parcels and General Supplies');
```

Comment:
Question: Do we want a dedicated table for this, or should we just create an `is_parcel_merchant` flag in `npc_types`?

Reply:
It is a good question and I can adjust as required.  I would certainly support the addition of the column to the npc table.

Reply:
How would we deal with the differences for expansions for the last name.  If pre RoF2, use General Supplies, and RoF2 with the addition of Parcels?  Any thoughts?

Reply:
I think the way they would handle this is a different NPC ID for that expansion of the game with a different lastname and the `is_parcel_merchant` flag set to `true`.

Reply:
Refactored to add the is_parcel_merchant column to db.  I added the updates to the db manifest for lastname, however these will also need to be added to source db once this PR is approved. (@joligario )  I did not duplicate the npc entries at this point and can do that as the expansion work progresses.